### PR TITLE
updated svm tests

### PR DESCRIPTION
**Pull request by rjbaraldi**
Had some other tests floating around. Let me know if they look ok.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -92,67 +92,51 @@ end
...
 
-#   @show @benchmark jtprod_residual!($fk, $xk, $v, $Jtvac)
-#   @show @benchmark jtprod_residual!($fad, $xk, $v, $Jtvac_ad)
+#   # @show @benchmark jtprod_residual!($fk, $xk, $v, $Jtvac)
+#   # @show @benchmark jtprod_residual!($fad, $xk, $v, $Jtvac_ad)
 
-#   @show norm(Jtvac - Jtvac_ad)
+#   @show norm(Jtvac - Jtvac_ad) < eps()
 
 # end
```
Comment (dpo) [inline test/runtests.jl:142]: Let's just remove all this commented-out code.

**Review by dpo — CHANGES_REQUESTED** [review]
Great, thanks! Just a minor comment.

File: test/runtests.jl (hunk 1 above)
Comment (rjbaraldi) [reply test/runtests.jl]: Sounds good.

**Comment by dpo** [comment]
Thanks. Branch needs to be rebased.

**Review by dpo — APPROVED** [review]
thanks!

**Pull request merged**